### DISCUSSION TITLE
Adding Hotkeys

### DISCUSSION THREAD

**Discussion by Chiro1314** [w-okada/voice-changer#996] [Ideas]
So hey guys. I m new around here. Non technical field lol. So yeah basically hotkeys because i tend to stream online and i had a thought that maybe it was possible to add the functionality of hotkey combinations, the voice changer would be a game changer for the channel points system on twitch. So yeah. I hope so this was the correct place to post such stuff around here.

**Comment by SmartDogo** [comment]
i would love to have a push to talk hotkey and toggle to mute

**Comment by R-Laine** [comment]
Please add this I really need push to talk, why do I need to press a button in the ui???
